demo/index.html

```diff
@@ -98,14 +106,30 @@
     <pre><code class="js">baguetteBox.run('.baguetteBoxOne');</code></pre>
 
     <div class="baguetteBoxOne gallery">
-        <a href="img/1-1.jpg" data-caption="Golden Gate Bridge"><img src="img/thumbs/1-1.jpg" alt="Golden Gate Bridge"></a>
-        <a href="img/1-2.jpg" title="Midnight City"><img src="img/thumbs/1-2.jpg" alt="Midnight City"></a>
-        <a href="img/1-3.jpg"><img src="img/thumbs/1-3.jpg" alt="Image 1.3"></a>
-        <a href="img/1-4.jpg"><img src="img/thumbs/1-4.jpg" alt="Image 1.4"></a>
-        <a href="img/1-5.jpg"><img src="img/thumbs/1-5.jpg" alt="Image 1.5"></a>
-        <a href="img/1-6.jpg"><img src="img/thumbs/1-6.jpg" alt="Image 1.6"></a>
-        <a href="img/1-7.jpg"><img src="img/thumbs/1-7.jpg" alt="Image 1.7"></a>
-        <a href="img/1-8.jpg"><img src="img/thumbs/1-8.jpg" alt="Image 1.8"></a>
+        <a href="img/1-1.jpg" data-caption="Golden Gate Bridge">
+            <img src="img/thumbs/1-1.jpg" alt="Golden Gate Bridge">
+        </a>
+        <a href="img/1-2.jpg" title="Midnight City">
+            <img src="img/thumbs/1-2.jpg" alt="Midnight City">
+        </a>
+        <a href="img/1-3.jpg">
+            <img src="img/thumbs/1-3.jpg" alt="Image 1.3">
+        </a>
+        <a href="img/1-4.jpg">
+            <img src="img/thumbs/1-4.jpg" alt="Image 1.4">
+        </a>
+        <a href="img/1-5.jpg">
+            <img src="img/thumbs/1-5.jpg" alt="Image 1.5">
+        </a>
+        <a href="img/1-6.jpg">
+            <img src="img/thumbs/1-6.jpg" alt="Image 1.6">
+        </a>
+        <a href="img/1-7.jpg">
+            <img src="img/thumbs/1-7.jpg" alt="Image 1.7">
+        </a>
+        <a href="img/1-8.jpg">
+            <img src="img/thumbs/1-8.jpg" alt="Image 1.8">
```

Comment:
Some time ago I read that meaningless `alt` attributes should be left empty:
https://davidwalsh.name/accessibility-tip-empty-alt-attributes
So I think you can remove that `Image 1.X` content with this PR.